hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -606,6 +606,9 @@ cdef class PureMarketMakingStrategy(StrategyBase):
             MarketBase market = self._market_info.market
             list buys = []
             list sells = []
+        # It is necessary to point buy and sell levels to order_levels in order to make the configurable on the fly
```

Comment:
You can do this in the order_levels property setter. Resetting the buy and sell levels here will be a problem for scripts.